### DISCUSSION TITLE
Env var not propagated to jobs

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17734] [Q&A]
Heya Team ,
I have environment variables to be used for dagster jobs which are created and ran by dagster pods in k8s. So I have the env var variable set properly on code location pod but its not getting populated to the dagster jobs. Any suggestion for the same ?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699273798310549?thread_ts=1699273798.310549&cid=C01U954MEER).*

**Comment by jamiedemaria** [comment]
Can you provide a code snippet of what you have tried so far?
